Solutions/Google Cloud Platform Firewall Logs/solutionMetadata.json

```diff
@@ -0,0 +1,20 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-gcpfirewalllogs",
+	"firstPublishDate": "2024-11-03",
+	"providers": [
+		"Google"
+	],
+	"categories": {
+		"domains": [
+			"DevOps",
+			"Security"
```

Comment:
domain should be not only security 

Please confirm on domain categories once

Reply:
Will be change to Security - Network instead